### PR TITLE
Development

### PR DESCRIPTION
**Pull request by rlancaster96**
version 0.1.0

Alignment with bwa-mem2, somatic variant calling with mutect2 and annotation with snpeff
This branch was used for processing samples for data presented on July 17, 2024.

### CODE REVIEW

File: workflows/calling/svc.nf [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env nextflow
+
+// Create queue channels (consumable)
+// will need to split up into tumor channel and normal channel, use regex for this
+tumor_ch = Channel.fromPath("${params.outdir}/aligned/duplicate_marked/*_T_*.bam")
+tumor_ch_bai = Channel.fromPath("${params.outdir}/aligned/duplicate_marked/*_T_*.bam.bai")
```
Comment (kellrott) [inline workflows/calling/svc.nf:6]: Requiring a specific naming pattern for input files is a little weird. It requires the user to rename files in order to use workflow

File: example/nf_channels_params_template.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+"container_fastqc" : "/path/to/singularity/image/fastqc.sif",
+"container_multiqc" : "/path/to/singularity/image/multiqc.sif",
+"container_trimmomatic" : "/path/to/singularity/image/trimmomatic.sif",
+"container_samtools" : "/path/to/singularity/image/samtools.sif",
+"container_gatk" : "/path/to/singularity/image/gatk.sif",
+"container_bwa" : "/path/to/singularity/image/bwa.sif",
+"container_snpeff" : "/path/to/singularity/image/snpeff.sif",
+"container_bcftools" : "/path/to/singularity/image/bcftools.sif",
+"filesuffix" : "bam or cram",
+"bucket_dir" : "/path/to/bam or cram/files",
```
Comment (kellrott) [inline example/nf_channels_params_template.json:11]: If this is for mutation calling, they should explicitly name the tumor/normal bams, rather then point at a directory and have the tumor/normal inferred from the name

File: example/nf_channels_params_template.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+"all_read_pairs" : "/path/to/fastq/files/*_R{1,2}*.fastq.gz",
```
Comment (kellrott) [inline example/nf_channels_params_template.json:12]: Why are the FASTQs included?

File: README.md [hunk 4]
```diff
@@ -80,58 +50,285 @@ nextflow info
...
+- snpeff https://quay.io/repository/ohsu-comp-bio/snpeff
+
+Because Docker cannot be easily used on Exacloud, images can be pulled using Singularity, which converts them to .sif images. To pull images:
+
+```
+## Load singularity module 
+module load /etc/modulefiles/singularity/current
+
+## set singularity cache directory variable; can live anywhere but gscratch is recommended.
+export SINGULARITY_CACHEDIR=/home/exacloud/gscratch/CEDAR/<user>/singularity/cache
```
Comment (kellrott) [inline README.md:74]: Let's try to abstract out specific directory names, eventually these instructions will be used by people not a OHSU, and the exacloud paths won't apply

**Review by elisabethgoldman — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (rlancaster96) [reply README.md]: fixed! generalized instructions for singularity use

File: example/nf_channels_params_template.json (hunk 3 above)
Comment (rlancaster96) [reply example/nf_channels_params_template.json:12]: We were using only 1 params file for both alignment and somatic variant calling, so variables used in the alignment are in there too.

**Pull request merged**